mcs/build/profiles/monotouch_watch.make

```diff
@@ -9,6 +9,5 @@ NO_THREAD_SUSPEND_RESUME=1
 NO_MULTIPLE_APPDOMAINS=1
 NO_PROCESS_START=1
 NO_MONO_SECURITY=1
-MONO_FEATURE_APPLETLS=
```

Comment:
this changes behavior, MONO_FEATURE_APPLETLS wasn't defined before on monotouch_watch but now it is (due to it inheriting PROFILE_APPLETLS=1 from monotouch_runtime.make)